pkg/virtctl/offlinevm/offlinevm_test.go

```diff
@@ -5,10 +5,18 @@ import (
 
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
+	k8smetav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"kubevirt.io/kubevirt/pkg/api/v1"
+	"kubevirt.io/kubevirt/pkg/kubecli"
+	"github.com/golang/mock/gomock"
+	"k8s.io/apimachinery/pkg/types"
 )
 
 var _ = Describe("OfflineVirtualMachine", func() {
 
+	var ovmInterface *kubecli.MockOfflineVirtualMachineInterface
+	var ctrl = gomock.NewController(GinkgoT())
```

Comment:
Can you always create a new instance in the `BeforeEach` block? In the `AfterEach` block you will then have to call `ctrl.Finish()`. If `Finish` is not called, the mocks would return the expected objects, but final checks, if expected calls were triggered at all are not performed.